docker-compose.yml

```diff
@@ -222,6 +184,13 @@ services:
     cap_add: [ 'NET_ADMIN', 'NET_RAW' ]
     restart: on-failure
 
+  browserless:
+    build:
+      context: ./print/browserless
+    container_name: 'ecamp3-browserless'
+    ports:
+      - '3010:3000'
+
 volumes:
-  db-data:
-  db-data-postgres:
+  db-data: null
```

Comment:
is this necessary?

Reply:
Strong opinion from VSCode. If it annoys, I can try to disable the formatter